### ISSUE TITLE
Générateur ne fonctionne pas

### ISSUE THREAD

**Issue by ValentinGratz** [Poloin34/ZelfGenerateRib#1]
Bonjour,

Le générateur "ne fonctionne pas", je dis juste par cela, que je rempli bien les informations sur la page pour générer le pdf rib. 
Et une fois le rib télécharger, je suis "obligé" de le re-remplir directement dessus. 

Il ne prérempli pas avec les info que j'ai fourni dans le formulaire.

**Comment by Poloin34** [comment]
Exact. Le commit du 29 Janvier prévient qu'il ne faut plus utiliser le générateur.

Et ceux du 30 janvier le sabote. Je ne recommande plus les services ZELF et ne les supporte plus non plus.
